optuna/multi_objective/_hypervolume/base.py

```diff
@@ -4,11 +4,46 @@
 
 
 class BaseHypervolume(object, metaclass=abc.ABCMeta):
-    """Base class for hypervolume calculators"""
+    """Base class for hypervolume calculators
+
+        .. note::
+            In Optuna, this class is used for computing the hypervolume of objective points.
+            In other words, the coordinate of each point represents the one of the multi-objective
+            function's values.
+
+        .. note::
+            We assume that the each objective function to be minimized. If you use some objectives
+            to be maximized, please transform inputs to this class's `compute` method for the
+            minimization problem. For example,
+
+            .. code::
```

Comment:
Nice that you added an example. `.. code::` probably suffices as it's only intended for developers but how about leaving a TODO to turn it into `.. testcode::` so that we can make sure that it's up to date (or at least does not yield a syntax error)?